### PR TITLE
+rx

### PR DESCRIPTION
**Pull request by mfts**
`rx` is an extensible, modern and minimalist pixel editor implemented in rust

### CODE REVIEW

File: projects/crates.io/rx/package.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  strip: /v/
+
+build:
+  dependencies:
+    rust-lang.org/cargo: '*'
+  script:
+    cargo install --locked --path . --root {{prefix}}
+
+test:
+  rx -h
```
Comment (jhheider) [inline projects/crates.io/rx/package.yml:19]: Can we add a better test? -h doesn't always prove the binary will run.

Comment (jhheider) [reply projects/crates.io/rx/package.yml:19]: I tried
```yaml
test:
  dependencies:
    curl.se: '*'
  script: |
    rx -h
    curl http://www.libpng.org/pub/png/PngSuite/basn6a08.png >test.png
    rx -u $FIXTURE test.png
  fixture: |
    e test.png
    p/clear
    p/add #ff0000
    w test2.png
    q
```
but it a) opens a window, and b) won't quit from init. So, I think we might be stuck.

**Comment by jhheider** [comment]
For some reason I merged this, even though it doesn't build (why?).

Requires X11 libraries. Those could be packaged, though it's going to be a bear. But feel free to try it, if you want, @mfts .

**Comment by mfts** [comment]
I was already wondering why it got merged so quickly. Yes, X11 is gonna be a big one. I'll give it a try.

**Pull request merged**